src/lib/components/Swap/index.tsx

```diff
@@ -12,7 +14,38 @@ import Settings from './Settings'
 import SwapButton from './SwapButton'
 import Toolbar from './Toolbar'
 
-export default function Swap() {
+interface DefaultTokenAmount {
```

Comment:
How come we dont just use TokenAmount from SDK? In my refactor all the input and output is changed to represent this - so maybe best to address in #3081

Reply:
These defaults are simpler interfaces passed as a prop when setting up the widget, so they don't need to adhere to the full TokenAmount interface.